### PR TITLE
Remove nft transferlist from transactions list endpoint

### PR DESCRIPTION
**Pull request by Nana-EC**
**Detailed description**:
It's good practice to not overload the list endpoints with finer details of component objects.
The need to pull these finer details results in multiple db joins that reduce performance on REST API calls

- Remove `nft_transfer_list` from `api/v/transactions` response date.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:
The list is still included in the `api/v/transactions/:id` response and customers should pull from there

**Checklist**

- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/transactions.js [hunk 1]
```diff
@@ -156,6 +159,10 @@ const createTransferLists = (rows) => {
         valid_duration_seconds: utils.getNullableNumber(row.valid_duration_seconds),
         valid_start_timestamp: utils.nsToSecNs(validStartTimestamp),
       };
+
+      if (includeNfts) {
```
Comment (steven-sheehy) [inline hedera-mirror-rest/transactions.js:163]: `includeNfts` is redundant as you could just check if row contains `nft_transfer_list`

**Review by steven-sheehy — APPROVED** [review]
One simplification noted

**Review by ijungmann — APPROVED** [review]

**Review by xin-hedera — APPROVED** [review]
LGTM

I'll merge the changes to my PR later.

File: hedera-mirror-rest/transactions.js (hunk 1 above)
Comment (Nana-EC) [reply hedera-mirror-rest/transactions.js:163]: Can add this in the nft history PR. Wanted to be simple here since we're cherry picking late in the game

**Pull request merged**